static/app/views/eventsV2/utils.tsx

```diff
@@ -437,7 +437,7 @@ function generateExpandedConditions(
 }
 
 type FieldGeneratorOpts = {
-  organization: Organization;
+  organization: Organization | OrganizationSummary;
```

Comment:
The `Organization` type is just `Organization & {some other stuff}`. Maybe we should just switch this to `OrganizationSummary` if that's all we need. I removed `Organization` locally and did a types check and it passed.

I like `Organization` because it's a lot shorter than `OrganizationSummary` and it matches the variable name, but if `OrganizationSummary` is sufficient, maybe we should do that instead of `Organization | OrganizationSummary`